app/uk/gov/hmrc/gatekeeper/views/applications/ApplicationsView.scala.html

```diff
@@ -100,7 +101,8 @@ <h1 class="govuk-heading-l" id="applications-title">Applications</h1>
               Filter by Status
             </label>
             <select class="govuk-select full-width" id="status" name="status">
-              <option @if(status == None || status == Some("ALL")){selected} value="ALL" selected>All</option>
+              <option @if(status == None || status == Some("EXCLUDING_DELETED")){selected} value="EXCLUDING_DELETED" selected>All - Excluding deleted</option>
```

Comment:
selected attribute is in here twice - is that intended?

Reply:
Fixed